neofetch

```
@@ -53,7 +53,7 @@ get_distro() {
     [[ "$distro" ]] && return
 
     case "$os" in
-        "Linux")
+        "Linux" | "BSD" | "MINIX")
             if [[ "$(< /proc/version)" == *"Microsoft"* || "$(< /proc/sys/kernel/osrelease)" == *"Microsoft"* ]]; then
```

We need to suppress errors caused by this.